R/learner_polyreg_regr_polyFit.R

```diff
@@ -0,0 +1,66 @@
+#' @title Polynomial Regression Learner
+#' @name mlr_learners_regr.polyFit
+#'
+#' @description
+#' Polynomial regression.
+#' Calls [polyreg::polyFit()] from \CRANpkg{polyreg}.
+#'
+#' @section Initial parameter values:
+#' * `deg`: We have set this to 2, pretty arbitrarily.
+#' * `noisy`: We have set this to FALSE, to get no output on the console.
+#'
+#' @templateVar id regr.polyFit
+#' @template learner
+#' @template seealso_learner
+#' @export
+LearnerRegrPoly = R6Class("LearnerRegrPolyFit",
+  inherit = LearnerRegr,
+  public = list(
+    #' @description
+    #' Creates a new instance of this [R6][R6::R6Class] class.
+    initialize = function() {
+      param_set = ps(
+        deg = p_int(lower = 0, tags = c("train", "required")),
+        maxInteractDeg = p_int(lower = 0, tags = c("train")),
+        # we dont put param "use" here because these is determined by the task
+        return_xy = p_lgl(default = FALSE, tags = c("train")),
+        returnPoly = p_lgl(default = FALSE, tags = c("train")),
+        noisy = p_lgl(default = TRUE, tags = c("train"))
```

Comment:
```suggestion
        deg = p_int(lower = 0, tags = c("train", "required")),
        maxInteractDeg = p_int(lower = 0, tags = "train"),
        # we dont put param "use" here because these is determined by the task
        return_xy = p_lgl(default = FALSE, tags = "train"),
        returnPoly = p_lgl(default = FALSE, tags = "train"),
        noisy = p_lgl(default = TRUE, tags = "train")
```